packages/npm-package-json-lint-config/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@wordpress/npm-package-json-lint-config",
+  "version": "0.1.0",
+  "description": "WordPress npm-package-json-lint shareable config",
+  "author": "WordPress",
+  "license": "GPL-2.0-or-later",
+  "keywords": [
+    "wordpress",
+    "npm-package-json-lint",
+    "npm-package-json-lint-config"
+  ],
+  "homepage": "https://github.com/WordPress/packages/tree/master/packages/npm-package-json-lint-config/README.md",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/WordPress/packages.git"
+  },
+	"bugs": {
```

Comment:
Whitespace consistency.

Reply:
Thanks, fixed in https://github.com/WordPress/packages/pull/119/commits/963eae118d4d124a61472a5c2142e984d5d0c763